doc-api/source/installation.rst

```diff
@@ -0,0 +1,46 @@
+Installation
+============
+
+Although it's not possible to cover all possible ways to install PBxplore on any operating system,
+we try in this document to provide few guidelines regarding PBxplore setup.
+
+Supported Platforms
+-------------------
+
+Currently, `PBxplore` run with Python 2.7, 3.3 and 3.4 on Linux and Mac OS X.
+
+
+Dependencies
+------------
+
+To use `PBxplore`, the following libraries have to be installed.
+
+    `NumPy <http://numpy.scipy.org/>`_ >= 1.6.0
+        Numpy is the base package for numerical computing in python.
+
+Optionally, `PBxplore` can use the following packages:
+
+    `MDAnalysis <http://www.mdanalysis.org/>`_ >= 0.11
+        We use MDAnalysis for loading trajectories.
+        See the full supported list
+        `here <https://pythonhosted.org/MDAnalysis/documentation_pages/coordinates/init.html#id1>`_.
+
+    `Matplotlib <http://matplotlib.org/>`_ >= 1.4.0
+        All ploting functions use `matplotlib` package.
+
+    `Weblogo3 <http://weblogo.threeplusone.com/>`_ [#]_
+        `Weblogo3` is required to create logo from PB sequences.
+
+
+Installing PBxplore
+-------------------
+
+Once dependencies installed, the most straightforward way is to use `pip`:
+
+.. code-block:: bash
+
+    $ pip install pbxplore
+
```

Comment:
Being on Pypi does not make user able to install a package system wide 
without sudo. And you do not want users to mess with the system 
site-package directory... It would be great to have a general doc to 
link, but I could not find one.

On 15-12-15 12:02, Hub wrote:
>
> In doc-api/source/installation.rst 
> <https://github.com/pierrepo/PBxplore/pull/110#discussion_r47622700>:
>
> > +    `Matplotlib <http://matplotlib.org/>`_ >= 1.4.0
> > +        All ploting functions use `matplotlib` package.
> > +
> > +    `Weblogo3 <http://weblogo.threeplusone.com/>`_ [#]_
> > +        `Weblogo3` is required to create logo from PB sequences.
> > +
> > +
> > +Installing PBxplore
> > +-------------------
> > +
> > +Once dependencies installed, the most straightforward way is to use `pip`:
> > +
> > +.. code-block:: bash
> > +
> > +    $ pip install pbxplore
> > +
>
> I agree about the install from the source, few words to clone the 
> depot and install it.
> But, I'm not really convinced about the |--user| and the virtualenv. 
> These are "global" options/set up for all python packages so the user 
> would know about it. The main message here is to tell the package is 
> on Pypi
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/pierrepo/PBxplore/pull/110/files#r47622700>.
>